packages/ag-charts-website/src/features/docs/utils/filesData.ts

```diff
@@ -1,30 +1,35 @@
 import fs from 'fs/promises';
+import path from 'path';
 
 // NOTE: These imports can't be aliases because it is used by `astro.config.mjs`
 // and ts alias paths don't work there
 import { INTERNAL_FRAMEWORKS } from '../../../constants';
 import { getFolders } from '../../../utils/fs';
-import { type DocsPage, type InternalFrameworkExample, getContentRootFileUrl } from '../../../utils/pages';
-import { pathJoin } from '../../../utils/pathJoin';
+import {
+    type DocsPage,
+    type InternalFrameworkExample,
+    getContentRootFileUrl,
+    getExampleRootFileUrl,
+} from '../../../utils/pages';
 
 function ignoreUnderscoreFiles(page: DocsPage) {
     const pageFolders = page.slug.split('/');
     const pageName = pageFolders[pageFolders.length - 1];
     return pageName && !pageName.startsWith('_');
 }
 
-export const getExamplesPathUrl = ({ pageName }: { pageName: string }) => {
+export const getExamplesPath = ({ pageName }: { pageName: string }) => {
     const contentRoot = getContentRootFileUrl();
-    const sourceExamplesPath = pathJoin(contentRoot.pathname, 'docs', pageName, '_examples');
+    const sourceExamplesPath = path.join(contentRoot.pathname, 'docs', pageName, '_examples');
```

Comment:
Was there a reason you needed `path.join` here? It's fine for this file since it'll be run in node, but I created `pathJoin` so that it can be used on client side code as well node. I'm also in the process of updating how we use `pathJoin`, so wondering if there is something missing in it.

Reply:
I was err'ing on the side of simplifying things as I went, it didn't seem necessary for the extra overhead of potentially handling URLs when these paths are always on the local FS - I don't mind reverting if you have a preference that way though?

Reply:
Don't mind it using the node version for this specific file. I do want to figure out a better way to distinguish between which utils can only be used on node/build envs, but that's a task for another time.